app/models/rent.rb

```diff
@@ -0,0 +1,4 @@
+class Rent < ApplicationRecord
```

Comment:
All the fields are required! Remember to add the necessary validations to be sure the model is created with all the data!